.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request Message Format WG
+about: Suggest an idea or feature for Message Format
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
I think rephrasing "I'm always frustrated when [...]" to be something like "It is difficult to [...]" so that we de-emphasize the individual and try to keep emotions out of (what is probably) an objective technical issue.